### ISSUE TITLE
Air Temp/Humidity logging

### ISSUE THREAD

**Issue by billtyler** [MickaelGandecki/myfood#11]
I see the air temp and humidity on display in the bottom of the Temperature square.  But when I touch the square only the water temperature seems to be displayed in the screen.

Are the air temp and humidity not being tracked over time?

**Comment by MickaelGandecki** [comment]
The air temp/humidity values are recorded but not displayed as a graph on the touchscreen version.

You can see them through the local webserver: connect your RPi Wifi, then navigate on http://192.168.137.1:5000

**Comment by billtyler** [comment]
Ok I see it there.  Are the webapp and the uwp app supposed to be functionally equivalent?
Is the webapp for a screenless install?

**Comment by MickaelGandecki** [comment]
Yes, they are more or less functionally equivalent: in fact, it's rather complicated to have a touchless screen within a greenhouse. That's why we deploy our box without.
